src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -31,7 +31,7 @@
 use Symfony\Component\Messenger\MessageBusInterface;
 use Symfony\Component\Notifier\Notifier;
 use Symfony\Component\PropertyInfo\PropertyInfoExtractorInterface;
-use Symfony\Component\RateLimiter\TokenBucketLimiter;
+use Symfony\Component\RateLimiter\Strategy\TokenBucketLimiter;
```

Comment:
Given the interface is named `LimiterInterface`, shouldn't it be `Symfony\Component\RateLimiter\Limiter` ?

Currently all namespace named `Strategy` are related to a *StreatgyInterface. (`Lock/StrategyInterface`, `Workflow/SupportStrategyInterface`, `HttpClient/RetryStrategyInterface`, `Asset/VersionStrategyInterface`, `Messenger/RetryStrategyInterface`

Reply:
Maybe.. The `Symfony\Component\RateLimiter` namespace has so much "limit" or "rate limit" related names already.

Im not super passionate about any name. I just wanted to move these classes to a separate namespace.

Reply:
I can understand both arguments ... but I'd prefer `Strategy/`.

Reply:
:+1: for strategy as well.

A (last?) change to this component would involve renaming all "strategy" to "policy" to follow common rate limit terminology (or at least, that of the rate limit HTTP RFC).

Reply:
I agree with Wouter that having names more similar to the HTTP RFC would be a good thing.

Reply:
Done. I renamed namespace and config to "policy"